www/statusbar.js

```diff
@@ -171,4 +173,15 @@ channel.deviceready.subscribe(function () {
     
   
     onVisibilityChange();
+    
+    if(device.version.split(".")[0].indexOf(IOS_11_VERSION) == 0){
```

Comment:
Can we do this inside a function like the **onVisibilityChange**?

Reply:
Fixed!